python/test_file_generator.py

```diff
@@ -1717,12 +1634,12 @@ def v671_2_file(self):
 		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
 
 	def v672_1_file(self):
-		"""Set loan costs to random of -1 or blank."""
+		"""Set loan costs to 4.5."""
 		name = "v672_1.txt"
 		path = self.validity_path
 		ts = self.ts_df.copy()
 		lar = self.lar_df.copy()
-		lar.loan_costs = lar.loan_costs.map(lambda x: random.choice(("-1", "")))
+		lar.loan_costs = "4.5"
```

Comment:
Is this value producing failures? 

The FIG has: 1) Total Loan Costs must be a number greater than or equal to 0, NA, or Exempt, and cannot be left blank.

The test reads that any number >0 should pass.